### DISCUSSION TITLE
[Gatsby Cloud] How to setup multi-domain deployments?

### DISCUSSION THREAD

**Discussion by kara-todd** [gatsbyjs/gatsby#30116] [Help]
Does Gatsby cloud support multi-domain or environment deploys in some manner? For example, I have a site that will have multiple versions at different subdomains like `showcase-a.`, `showcase-b.` and I need to target and build the subdomains with specific content from the CMS.

For another site, we have a separate QA CMS version where we test API changes. So I would need to deploy to `qa.example.com` and to `www.example.com` with different environment variables.

**Comment by LekoArts** [comment]
Hi! For Gatsby Cloud questions please refer to this page: https://www.gatsbyjs.com/support/
Thanks!

**Reply by kara-todd** [reply]
Thanks for the reply @LekoArts.

Unfortunately, nothing on that page seems to address my question. I can certainly send a direct inquiry to support, but I thought it would be more helpful to have a public response incase other people have the same question.

Should I just reach out for private support instead?
